src/qt/bitcoin.cpp

```diff
@@ -251,6 +251,12 @@ BitcoinCore::BitcoinCore(interfaces::Node& node) :
 void BitcoinCore::handleRunawayException(const std::exception *e)
 {
     PrintExceptionContinue(e, "Runaway exception");
+    try {
+        m_node.appShutdown();
+    } catch (const std::exception& e) {
```

Comment:
`e` here shadows the function parameter in `BitcoinCore::handleRunawayException(const std::exception *e)`.